layers/meta-resin-raspberrypi/recipes-support/hostapp-update-hooks/files/99-rpi-bootloader

```diff
@@ -16,12 +16,11 @@ else
 fi
 
 new_part=$(findmnt --noheadings --canonicalize --output SOURCE $SYSROOT)
-blockdev=$(basename "$new_part")
-new_part_idx=$(cat "/sys/class/block/$blockdev/partition")
+new_part_label=$(blkid "$new_part" | awk '{print $2}')
```

Comment:
Please change the var name to something like "new_root_filesystem_label" as this is really a filesystem label and not a partition label

Reply:
OK, ... do I have to do something still?